manifests/init.pp

```diff
@@ -160,17 +160,19 @@
   $rhsm_path = '/rhsm'
   $rhsm_port = 443
 
+  $insights_path = '/redhat_access'
+
   ensure_packages('katello-debug')
 
   include certs::foreman_proxy
   Class['certs::foreman_proxy'] ~> Service['foreman-proxy']
 
   if $reverse_proxy_real {
     foreman_proxy_content::reverse_proxy { "rhsm-pulpcore-https-${reverse_proxy_port}":
-      url      => "${foreman_url}/",
-      port     => $reverse_proxy_port,
-      priority => '10',
-      before   => Class['pulpcore::apache'],
+      path_url_map => { '/' => "${foreman_url}${rhsm_path}" },
```

Comment:
This looks like a change where `/` used to point at `${foreman_url}`, it now points at `${foreman_url}${rhsm_path}`.